climpred/classes.py

```diff
@@ -1137,6 +1139,65 @@ def _verify(
                 res = _reset_temporal_axis(res, self._temporally_smoothed, dim='lead')
         return res
 
+    def bootstrap(
+        self,
+        metric='pearson_r',
+        comparison='e2o',
+        alignment='same_verifs',
+        sig=95,
+        dim='init',
+        resample_dim='member',
+        iterations=500,
+        pers_sig=None,
+        **metric_kwargs,
+    ):
+        """Bootstrap ensemble HindcastEnsemble with replacement according to
+        Goddard et al. 2013.
+
+        Args:
+            metric (str, default 'pearson_r'): Metric to apply for verification.
+            comparison (str, default 'e2o'): How to compare to the
+                observations/verification data. ('e2o' for ensemble mean to
+                observations/verification data. 'm2o' for each individual member to
+                observations/verification data).
+            alignment (str): which inits or verification times should be aligned?
+                - maximize/None: maximize the degrees of freedom by slicing ``hind`` and
+                ``verif`` to a common time frame at each lead.
+                - same_inits: slice to a common init frame prior to computing
+                metric. This philosophy follows the thought that each lead should be
+                based on the same set of initializations.
+                - same_verif: slice to a common/consistent verification time frame prior
+                to computing metric. This philosophy follows the thought that each lead
+                should be based on the same set of verification dates.
+            sig (int, default 95):
+                Significance level for uninitialized and initialized
+                comparison.
+            iterations (int, default 500): Number of resampling iterations for
+                bootstrapping with replacement.
+            pers_sig (int, default None):
+                If not None, the separate significance level for persistence.
+            **metric_kwargs (optional): arguments passed to `metric`.
+
```

Comment:
I think you need to add `dim` and `resample_dim` to the docstring here.

Reply:
added